packages/orbit-components/src/Collapse/index.tsx

```diff
@@ -64,14 +64,8 @@ const Collapse = ({
       data-test={dataTest}
       id={id}
     >
-      {/* eslint-disable-next-line jsx-a11y/click-events-have-key-events */}
-      <div
-        className="block w-full cursor-pointer"
-        onClick={handleClick}
-        role="button"
-        tabIndex={-1}
-        id={labelID}
-      >
+      {/* eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-noninteractive-element-interactions */}
+      <div className="block w-full cursor-pointer" id={labelID} onClick={handleClick} role="group">
```

Comment:
Could you please explain why we're using the `group` role here now?
In the end, the div is acting as a button, isn't it? It even has an onClick handler.

Reply:
IMO, the div should be kept as a button and the inner ButtonLink should be rendered as a div

Reply:
> the inner ButtonLink should be rendered as a div

I assume this would remove the role="button" quality, right?

Reply:
Yes, because the role="button" is only applied if the element is natively a button or if it has an onClick, which it doesn't, since the onClick is on the outer div